.github/workflows/api.yaml

```diff
@@ -10,8 +10,8 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.14.x
-    - name: GitHub Action for Homebrew
-      uses: artemnovichkov/action-homebrew@0.1.0
+    - name: Install Homebrew
+      run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)" 
```

Comment:
Can you merge master in and make sure that this isn't there anymore since we're using a binary?

Reply:
done